jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -25,7 +26,28 @@
 from jax import test_util as jtu
 from jax import numpy as jnp
 
+import os
+
 class JaxToTfTestCase(jtu.JaxTestCase):
+  @classmethod
+  def setUpClass(cls):
```

Comment:
I would move this whole thing to global scope. As it is now, every test case will setup its own collection and would overwrite the same file. If this is in the global scope, there is no need to restore the hooks.